src/pybind/mgr/dashboard/frontend/src/app/shared/components/bottom-links/bottom-links.component.html

```diff
@@ -0,0 +1,10 @@
+<ul class="list-inline">
+  <li class="list-inline-item p-2"
+      *ngFor="let docItem of docItems">
+    <cd-doc section="{{ docItem.section }}"
+            docText="{{ docItem.text }}"
+            noSubscribe="true"
+            i18n-docText></cd-doc>
+  </li>
+</ul>
+
```

Comment:
After seeing how lightweight this new component is (basically a thin wrapper around the docs one), wouldn't it make more sense to simply embed the code into the login page or are we planning to embed it somewhere else?

Reply:
I dont think these links are that useful to embed it into some other components. Atleast I am not seeing anything of that sort. So it make much more sense to just embed the code into the login page and remove the component altogether.

Reply:
Removed the component and embedded it along with the login-layout.component.